notebooks/backwards_compatability.ipynb

```diff
@@ -0,0 +1,985 @@
+{
```

Comment:
<p>Not sure what is prepackaged iris testing here, it will be just deployed using old server image so it's just </p><p>"old operator + "old sklearn server"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/2574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>